python-project-template/pyproject.toml.jinja

```diff
@@ -38,6 +38,9 @@ dev = [
 {% elif preferred_linter == 'black' %}
     "black", # Used for static linting of files
 {% endif %}
+{% if mypy_type_checking != 'none' %}
```

Comment:
nit: @OliviaLynn has introduced the syntax `{%- ...` to remove the blank whitespaces if the conditional isn't met, like you see here. https://github.com/lincc-frameworks/python-project-template/pull/99/files#diff-617554b3505e1027c67cb3b2a782bfadb0bca98bd16329e9112e006aa4b33801

It would be nice to include that :)

Reply:
Oh that's nice, added :)